Readme.md

```diff
@@ -1,21 +1,69 @@
-# Lombiq <add project name here>
+# Lombiq PowerShell Analyzers
 
 
 
 ## About
 
-Add a general overview of the project here. Keep or remove the OSOCE note below as necessary.
+Powershell static code analysis via [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) and [our configuration for it](PSScriptAnalyzerSettings.psd1).
 
 Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!
 
 
 ## Documentation
 
-Add detailed documentation here. If it's a lot of content then create documentation pages under the *Docs* folder and link pages here.
+### Pre-requisites
 
+The *PSScriptAnalyzer* module must be installed. If it's not installed, then running the script in `pwsh` will output no results and running it in `powershell` will throw an exception. Follow the steps [here](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/overview?view=ps-modules#installing-psscriptanalyzer).
```

Comment:
> Can't we install this automatically if it's not installed?

We can attempt it, but it won't always work. It definitely won't work on Windows PowerShell becuase you need to be admin to install modules. I have added some nested try-catches and wrote a more informative error message to clarify that situation.

> Also, it would be better to have consistent behavior in both PS editions.

I have uninstalled the module on pwsh and it didn't reproduce the different behavior I described. So I guess my machine was just messed up. (is this the rare reverse "it works on my machine"? :D)

Reply:
Yep, that works.

I think the lack of module is an exception in every case, but it doesn't actually matter. I'm now checking for an exact version using `Get-InstalledModule` before the module is used in any way and we only invoke the cmdlet when it's installed.